tests/tests_geomstats/test_hyperboloid.py

```diff
@@ -14,11 +14,13 @@
 class TestHyperboloid(LevelSetTestCase, metaclass=Parametrizer):
     skip_test_extrinsic_after_intrinsic = True
     skip_test_projection_belongs = True
+    skip_test_extrinsic_ball_extrinsic_composition = True
```

Comment:
Why did these skips appear? was something working previously, but not anymore with the new version? If so, what is the plan to recover the previous behavior?

Reply:
These tests assume `Hyperboloid` can receive `default_coords_type="intrinsic"`, which is not the case anymore. I'm keeping the tests to check later what is missing now and how could this behavior be replicated in a different way.